snippets/Google.Datastore.V1Beta3.Snippets/DatastoreClientSnippets.cs

```diff
@@ -334,19 +335,15 @@ public void UpdateEntity()
             string projectId = _fixture.ProjectId;
             Key key = _fixture.LearnDatastoreKey;
 
-            // TODO: Fix transaction handling
             // Snippet: UpdateEntity
             DatastoreClient client = DatastoreClient.Create();
-            ByteString transaction = client.BeginTransaction(projectId).Transaction;
-
-            LookupResponse response = client.Lookup(
-                projectId,
-                new ReadOptions { ReadConsistency = ReadConsistency.Strong, Transaction = transaction },
-                new[] { key });
-
-            Entity entity = response.Found[0].Entity;
-            entity["priority"] = 5;
-            client.Commit(projectId, Mode.Transactional, transaction, new[] { entity.ToUpdate() });
+            using (DatastoreTransaction transaction = client.CreateDatastoreTransaction(projectId))
+            {
+                Entity entity = transaction.Get(key);
```

Comment:
Have renamed the method in DatastoreClient to `LookupAll` to avoid the collision with the API call. It can be called Lookup again in the wrapper.